src/types/coord.rs

```diff
@@ -31,22 +30,6 @@ where
     pub m: Option<T>,
 }
 
-impl<T> fmt::Display for Coord<T>
```

Comment:
Though it seems unlikely anyone is using it, can we leave it as it was and deprecate it first, for sake of avoiding a breaking change?

Reply:
<img width="565" alt="image" src="https://github.com/user-attachments/assets/65fadbf4-abcd-4dd2-b4bc-baf2cc61b215">
is it possible to deprecate a trait impl? Seems not. https://users.rust-lang.org/t/deprecate-an-implementation/90350

Reply:
Darn. Sorry for the noise!

Reply:
Well we could just leave this trait impl in even though it's no longer used